### PR TITLE
clipbuzz: init at 2.0.0

### PR DESCRIPTION
**Pull request by McSinyx**
###### Motivation for this change
clipbuzz is a fork of clipnotify to be used with my clipboard manager.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/clipbuzz/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  buildInputs = [ libX11 libXfixes ];
+
+  preBuild = "export HOME=$TMPDIR";
+  installPhase = ''
+    zig build -Drelease-safe -Dcpu=baseline --prefix $out install
+  '';
+
+  meta = with lib; {
+    description = "Buzz on new X11 clipboard events";
+    inherit (src.meta) homepage;
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/clipbuzz/default.nix:24]: Please set the homepage explicitly. I am not going to copy the url together from multiple variables.

File: pkgs/tools/misc/clipbuzz/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    owner = "~cnx";
+    repo = pname;
+    rev = version;
+    sha256 = "1qn98bwp7v7blw4v0g4pckgxrky5ggvq9m0kck2kqw8jg9jc15jp";
+  };
+
+  nativeBuildInputs = [ zig ];
+  buildInputs = [ libX11 libXfixes ];
+
+  preBuild = "export HOME=$TMPDIR";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/clipbuzz/default.nix:17]:
```suggestion
  preBuild = ''
    export HOME=$TMPDIR
  '';
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/tools/misc/clipbuzz/default.nix (hunk 1 above)
Comment (McSinyx) [reply pkgs/tools/misc/clipbuzz/default.nix:24]: Sorry, I saw this in other plan and thought it was conventional :smile:

Comment (SuperSandro2000) [reply pkgs/tools/misc/clipbuzz/default.nix]: When I get around I am probably removing that from all files in nixpkgs. It is a nice little shortcut but while reviewing it is pretty tedious to visit the website.

**Pull request merged**